mydoc/mydoc_cmip6_validator.md

```diff
@@ -54,14 +54,14 @@ where:
 
 PrePARE will verify that all attributes in the input file are present and conform to CMIP6 for publication into ESGF.  We also recommand running the python program [cfchecker](https://pypi.python.org/pypi/cfchecker) created by the University of Reading in the UK to confirm that your file is CF-1 compliant.
 
-  * In order to validate all CMIP6 required attributes by PrePARE,  a [Controlled Vocabulary file](https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_CV.json) is read by the program where a JSON dictionnary called  ["required_global_attributes"](https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_CV.json#L3) point to a list of strings.  Each element of that list corresponds to a global attribute.  
-  * PrePARE can also use regular expressions to validate the value of the some global attributes.  Here is an [example](https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_CV.json#L6343-L6344) used for variant_label.
+  * In order to validate all CMIP6 required attributes by PrePARE,  a [Controlled Vocabulary file](https://github.com/PCMDI/cmip6-cmor-tables/blob/main/Tables/CMIP6_CV.json) is read by the program where a JSON dictionnary called  ["required_global_attributes"](https://github.com/PCMDI/cmip6-cmor-tables/blob/a46dcbf17ec5e11af23dc2d55107f5e52afbcade/Tables/CMIP6_CV.json#L3) point to a list of strings.  Each element of that list corresponds to a global attribute.  
+  * PrePARE can also use regular expressions to validate the value of the some global attributes.  Here is an [example](https://github.com/PCMDI/cmip6-cmor-tables/blob/a46dcbf17ec5e11af23dc2d55107f5e52afbcade/Tables/CMIP6_CV.json#L11554-L11555) used for variant_label.
 
-  * Institutions and institution_ids need to be registered into a list.   PrePARE will only accept institutions which have been pre-registered for CMIP6 publication into ESGF.   Click [here](https://github.com/PCMDI/cmip6-cmor-tables/blob/master/Tables/CMIP6_CV.json#L65) for the list of institutions.  If you wish to register your institution write to the [cmor mailing list](mailto:cmor@listserv.llnl.gov).
+  * Institutions and institution_ids need to be registered into a list.   PrePARE will only accept institutions which have been pre-registered for CMIP6 publication into ESGF.   Click [here](https://github.com/PCMDI/cmip6-cmor-tables/blob/a46dcbf17ec5e11af23dc2d55107f5e52afbcade/Tables/CMIP6_CV.json#L74) for the list of institutions.  If you wish to register your institution write to the [cmor mailing list](mailto:cmor@listserv.llnl.gov).
```

Comment:
Registering an institution will require engagement with the appropriate project_CVs repo, e.g. CMIP6Plus_CVs, CMIP7_CVs, input4MIPs_CVs, and obs4MIPs-cmor-tables (which will migrate to obs4MIPs_CVs once we have all the tables etc in place)